.github/workflows/integration-tests.yml

```diff
@@ -463,20 +439,10 @@ jobs:
 
           mkdir -p ~/.ccache
           du -h -d 1 ~/.ccache
-      - # If we're on branch `main`, save the ccache Triton compilation artifacts
-        # to the cache so they can be used by other (non-main) CI runs.
-        #
-        # (It wouldn't be a problem to save the cache on every run, because github
-        # evicts cache entries LRU, but maybe this saves a bit of time in CI.)
-        name: Save ccache and Triton compilation artifacts to cache
-        if: github.ref == 'refs/heads/main'
-        uses: actions/cache/save@v4
-        with:
-          path: |
-            ~/.triton/cache
-            ~/.ccache
-          key: triton-artifacts-${{ runner.os }}-${{ runner.arch }}-${{ env.RUNNER_TYPE }}-llvm-${{ steps.cache-key.outputs.llvm }}-${{ steps.cache-key.outputs.datetime }}
-      - name: Clean up caches
+      -
+      # Build cache right now takes a long time to save on MI GPU CI machines. Disabled for now.
+      #- *save-build-artifacts-step
+        name: Clean up caches
         run: |
           rm -rf ~/.triton/cache
```

Comment:
I think you're just missing `rm -rf ~/.ccache` here. Currently the AMD ccache directory is 50 times bigger than the NVIDIA one so no wonder it's taking longer.

Reply:
Let's try to remove the cache and check the restore time

Reply:
The build artifact cache doesn't buy us much benefits for test execution (which I assume is the purpose) though?

* w/: https://github.com/triton-lang/triton/actions/runs/12208440564/job/34076098946?pr=5363
* w/o: https://github.com/triton-lang/triton/actions/runs/12249730219/job/34171535982?pr=5384

 case | python test | regression test | proton test
--- | --- | --- | ---
w/ | 5m53s | 30s | 11s
w/o | 6m1s | 29s | 11s

So feels to me disabling it is fine.

Reply:
See: https://github.com/triton-lang/triton/actions/runs/12249730219/job/34171535982?pr=5384#step:6:1

Reply:
we are paying the cost of 6m+ for downloading/uploading. I guess we can trim down the amount of data being transferred over network but still would likely be some minutes there.

Reply:
It will be 50x smaller, so around 6 seconds...

Reply:
I've opened #5391, so we can test it out there.